packages/regenerator-runtime/runtime-module.js

```diff
@@ -1,9 +1,6 @@
 // This method of obtaining a reference to the global object needs to be
 // kept identical to the way it is obtained in runtime.js
-var g =
-  typeof global === "object" ? global :
-  typeof window === "object" ? window :
-  typeof self === "object" ? self : this;
+var g = (function { return this })();
```

Comment:
An anonymous and parameter-less function?? Think you need a `()`, as the tests will surely tell you.